pipelines/treatment/monitoramento/constants.py

```
@@ -201,7 +204,7 @@ class constants(Enum):  # pylint: disable=c0103
     )
 
     MONITORAMENTO_TEMPERATURA_TEST = DBTTest(
-        model="temperatura_inmet aux_viagem_temperatura aux_veiculo_falha_ar_condicionado veiculo_regularidade_temperatura_dia",  # noqa
+        model="teste_completude_temperatura temperatura_alertario aux_viagem_temperatura aux_veiculo_falha_ar_condicionado veiculo_regularidade_temperatura_dia temperatura",  # noqa
```

Ruff (0.14.2)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)